unit_tests/CommentUnitTests.d

```diff
@@ -0,0 +1,243 @@
+/**
+ * Copyright: Copyright (c) 2016 Wojciech Szęszoł. All rights reserved.
+ * Authors: Wojciech Szęszoł
+ * Version: Initial created: May 21, 2016
+ * License: $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0)
+ */
+
+import std.stdio;
+import Common;
+import dstep.translator.Translator;
+
+// Empty file translates to empty file.
+unittest
+{
+    assertTranslates(
+q"C
+C",
+q"D
+D", true);
+}
+
+// Test disabled comments.
+unittest
+{
+    Options options;
+    options.language = Language.c;
+    options.enableComments = false;
+
+    assertTranslates(
+q"C
+    /* Disabled comments. */
+C",
+q"D
+D", options, false);
+}
+
+// Test single comment.
+unittest
+{
+    assertTranslates(
+q"C
+/* Enabled comments. */
+C",
+q"D
+/* Enabled comments. */
+D", true);
+}
+
+// Test header comment handling.
+unittest
+{
+	assertTranslates(
+q"C
+/* Header comment. */
+
+/* Comment before variable. */
+int variable;
```

Comment:
The header comment is the comment in the first line, if there is no such comment (e.g. fist line empty) there is no header comment. 

> If /* Comment before variable. */ is removed, is /* Header comment. */ still a header comment?

Yes.